.gitmodules

```diff
@@ -17,3 +17,6 @@
 	path = 3rdparty/OpteeCalls
 	url = https://github.com/ms-iot/OpteeCalls.git
 	branch = master
+[submodule "3rdparty/json11"]
+	path = 3rdparty/json11
```

Comment:
Ok for the moment, but in the future, I'd much prefer our projects use https://github.com/akheron/jansson instead.  That's because it's used by the GitHub jose library, and I previously ported the jose library and jansson to work in an enclave, for the OTrP project.

Reply:
Ok, will keep in mind. Dropping for now from here, as this was only needed by oe_policy_tool, which is moved to a different repo.